tests/test_filtered_wells.py

```diff
@@ -35,14 +35,29 @@
 def test_incorrect_filter(test_client: TestClient):
     """Expect a 422 response when an incorrect filter is given."""
 
-    response = test_client.get("/pacbio/wells?qc_status=thisdoesntexist")
+    response = test_client.get(
+        "/pacbio/wells?page_size=1&page_number=1&qc_status=thisdoesntexist"
+    )
     assert response.status_code == 422
 
-    response = test_client.get("/pacbio/wells?weeks=-1")
+    response = test_client.get("/pacbio/wells?page_size=1&page_number=-1")
+    assert response.status_code == 422  # positive value is expected
+    response = test_client.get("/pacbio/wells?page_size=-11&page_number=1")
     assert response.status_code == 422  # positive value is expected
 
+    response = test_client.get("/pacbio/wells")
+    assert response.status_code == 422  # page size and number should be given
+
+    response = test_client.get("/pacbio/wells?page_number=1")
+    assert response.status_code == 422  # page size should be given
 
-def assert_filtered_inbox_equals_expected(response, expected_data):
+    response = test_client.get("/pacbio/wells?page_size=1")
+    assert response.status_code == 422  # page number should be given
```

Comment:
After a fashion, these are testing pydantic and fastapi. Do we need these 422-type tests?